lib/cn/cn.c

```diff
@@ -211,23 +211,22 @@ struct perfc_set *
 cn_get_perfc(struct cn *cn, enum cn_action action)
 {
     switch (action) {
+    case CN_ACTION_NONE:
+    case CN_ACTION_END:
```

Comment:
Anybody else want to get rid of NONE and END?

Reply:
It has crossed my mind more than once!!!

Reply:
Do it!

Reply:
I got rid of CN_ACTION_END, but CN_ACITON_NONE is used as a default state, which makes sense, so I kept it...